### PR TITLE
Renaming Java 18.3 to Java 10

### PR DESCRIPTION
**Pull request by JasonFengJ9**
Renaming `Java 18.3` to `Java 10`

Updated `JPP` tags accordingly;
Updated `J2SE_` Java version and build shape as well. 

Reviewer @Peter-Shipton @Tobi-Ajila 
FYI: @daniel-heidinga 

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: runtime/oti/j2sever.h [hunk 1]
```diff
@@ -36,13 +36,13 @@
 #define J2SE_17  0x1700
 #define J2SE_18  0x1800
 #define J2SE_19  0x1900
-#define J2SE_2018_3  0xB700            /* 0xB7 is 183 which refers to Java 18.3 */
+#define J2SE_1A  0x1A00
```
Comment (pshipton) [inline runtime/oti/j2sever.h]: Please comment that this means Java 10

Comment (pshipton) [reply runtime/oti/j2sever.h]: or we could use a different constant like J2SE_v10

File: runtime/oti/j2sever.h [hunk 2]
```diff
@@ -67,13 +67,13 @@
...
 #else
-	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_B1803
+	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_B1000
 #endif
 #define J2SE_SHAPE_SUN     		0x10000
 #define J2SE_SHAPE_B136    		0x40000
 #define J2SE_SHAPE_B148    		0x50000
 #define J2SE_SHAPE_B165    		0x60000
-#define J2SE_SHAPE_B1803		0x70000
+#define J2SE_SHAPE_B1000		0x70000
```
Comment (pshipton) [inline runtime/oti/j2sever.h]: We could change the name to something like J2SE_SHAPE_V10. In the future if build levels are also required then we would add J2SE_SHAPE_V10_B24

Comment (pshipton) [reply runtime/oti/j2sever.h]: We could also rename the other shapes like J2SE_SHAPE_B136 -> J2SE_SHAPE_V9_B136 and J2SE_SHAPE_B165 -> J2SE_SHAPE_V9

File: runtime/oti/j2sever.h (hunk 1 above)
Comment (JasonFengJ9) [reply runtime/oti/j2sever.h]: Changed it to `J2SE_v10` and added a comment to indicate it is `Java 10`.

File: runtime/oti/j2sever.h (hunk 2 above)
Comment (pshipton) [reply runtime/oti/j2sever.h]: > I feel these Java 9 renaming could be done in a separate cleanup PR instead.
Agreed

Comment (pshipton) [reply runtime/oti/j2sever.h]: J2SE_SHAPE_B1000 should be renamed as part of this PR, are you doing that?

Comment (JasonFengJ9) [reply runtime/oti/j2sever.h]: Updated now, pls have another look.

**Pull request merged**